test/unit/modules/multisignatures.js

```diff
@@ -911,4 +968,252 @@ describe('multisignatures', () => {
+				stubs.logic.account.getMultiSignature;
+			library.db.multisignatures.getMemberPublicKeys =
+				stubs.getMemberPublicKeys;
+			get('modules').accounts.getAccounts = stubs.modules.accounts.getAccounts;
+			get('modules').accounts.generateAddressByPublicKey =
+				stubs.modules.accounts.generateAddressByPublicKey;
+			done();
+		});
+
+		it('should accept address as parameter', done => {
+			self.getGroup(validAccount.address, (err, scopeGroup) => {
+				expect(library.logic.account.getMultiSignature).to.be.calledWith({
+					address: validAccount.address,
+				});
+				expect(scopeGroup.address).to.equal(validAccount.address);
+				done();
+			});
+		});
+
+		it('should fail if getMultiSignature function returns an error', done => {
+			library.logic.account.getMultiSignature = sinonSandbox
+				.stub()
+				.callsFake((filters, cb) => {
+					cb('Err', null);
+				});
+			self.getGroup(validAccount.address, (err, scopeGroup) => {
+				expect(err).to.equal('Err');
+				expect(scopeGroup).to.not.exist;
+				done();
+			});
+		});
+
+		it('should fail if given address does not return an account', done => {
+			library.logic.account.getMultiSignature = sinonSandbox
+				.stub()
+				.callsFake((filters, cb) => {
+					cb(null, null);
+				});
+			self.getGroup('', (err, scopeGroup) => {
+				expect(err).to.be.an.instanceof(ApiError);
+				expect(err.message).to.equal('Multisignature account not found');
+				expect(err.code).to.equal(errorCodes.NOT_FOUND);
+				expect(scopeGroup).to.not.exist;
+				done();
+			});
+		});
+
+		it('should return a group if getMultiSignature function returns a valid multisig account', done => {
+			self.getGroup(validAccount.address, (err, scopeGroup) => {
+				expect(err).to.not.exist;
+				expect(scopeGroup).to.have.property('address');
+				expect(scopeGroup).to.have.property('publicKey');
+				expect(scopeGroup).to.have.property('secondPublicKey');
+				expect(scopeGroup).to.have.property('balance');
+				expect(scopeGroup).to.have.property('unconfirmedBalance');
+				expect(scopeGroup).to.have.property('min');
+				expect(scopeGroup).to.have.property('lifetime');
+				expect(scopeGroup).to.have.property('members');
+				done();
+			});
+		});
+
+		it('should return group members if getMemberPublicKeys function returns an array of member account keys', done => {
+			library.db.multisignatures.getMemberPublicKeys = sinonSandbox
+				.stub()
+				.callsFake(() => Promise.resolve(['key1']));
+
+			const member = {
+				address: 'address',
+				publicKey: 'publicKey',
+				secondPublicKey: 'secondPublicKey',
+			};
+
+			stubs.modules.accounts.getAccounts = sinonSandbox
+				.stub()
+				.callsFake((param1, param2, cb) => cb(null, [member]));
+
+			self.getGroup(validAccount.address, (err, scopeGroup) => {
+				expect(err).to.not.exist;
+				expect(get('modules').accounts.getAccounts).to.be.calledWith({
+					address: ['address1'],
+				});
+				expect(scopeGroup.members)
```

Comment:
please also test with more than 1 member

Reply:
I refactored this test to have multiple members instead of adding another one.